src/pages/api/download/models/[modelVersionId].ts

```diff
@@ -137,6 +144,10 @@ export default PublicEndpoint(
 
     // Track download
     try {
+      if (!fileResult.isDownloadable) {
+        throw new Error('File not downloadable. Either moderator or owner download. Ignore.');
```

Comment:
This message is a bit confusing, let's try this, what do you think?
```suggestion
        throw new Error('File not downloadable. Either a moderator or the resource owner disabled downloads for this version.');
```